apollo-adminservice/pom.xml

```diff
@@ -175,6 +175,16 @@
 				</dependency>
 			</dependencies>
 		</profile>
+		<profile>
+			<id>zookeeper-discovery</id>
```

Comment:
Would it be better to keep the dependency handling of spring-cloud-starter-zookeeper-discovery the same as spring-cloud-starter-consul-discovery so that users don't have to care about compile-time configuration and the official distribution package can use zk? It should be noted that when dealing with nacos-related dependencies, the profile optional dependency was used because the nacos dependency would conflict with eureka

Reply:
zk discovery dependency has handled as sams as consul